core/tauri-runtime-wry/src/lib.rs

```diff
@@ -2590,14 +2625,16 @@ fn handle_user_message<T: UserEvent>(
       }
     }
     Message::Webview(id, webview_message) => match webview_message {
-      WebviewMessage::EvaluateScript(script) => {
+      WebviewMessage::EvaluateScript(script, tx, span) => {
+        let _guard = span.entered();
         if let Some(WindowHandle::Webview { inner: webview, .. }) =
           windows.borrow().get(&id).and_then(|w| w.inner.as_ref())
         {
           if let Err(e) = webview.evaluate_script(&script) {
             debug_eprintln!("{}", e);
           }
         }
+        tx.send(()).unwrap();
```

Comment:
I am not aware of the performance cost that we would get from this extra logic so I think we should add all tracing logic behind a feature flag (disabled by default)

Reply:
it just has to wait for the message to arrive on the main thread so it shouldn't be that bad but we can do the flag

Reply:
yeah ideally the flag should be added for those who want to squeeze each last piece of performance

Reply:
@amrbashir @JonasKruckenberg I didn't test yet but I pushed so you can take a look.. tbh I think it bloats the code a lot.

Reply:
it does bloat it a bit but not that much IMO